tools/cli/src/helpers/version.ts

```diff
@@ -0,0 +1,12 @@
+import { AsyncResult, parseJSON, readFile } from '@backtrace/sourcemap-tools';
+import path from 'path';
+
+interface PackageJson {
+    readonly version: string;
+}
+
+export function loadVersion() {
+    return AsyncResult.equip(readFile(path.join(__dirname, '../../package.json')))
```

Comment:
The approach you took here is technically correct, however it's super easy to make a mistake with it. Let's say now we're changing for any reason the directory structure. It's a trivial change and nothing should break, right? 

with this simple change, we're introducing an error with missing package.json version. 

Instead: can we verify if:
`path.join(require.main.path, '..', 'package.json')` exists? The approach is better, because the structure of the entry point of the package shouldn't change or can change but definitely less often. 

what do you think?

Reply:
done